jenkins/basic-aptly-build-job.xml

```diff
@@ -7,14 +7,24 @@
   <properties />
   <definition class="org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition" plugin="workflow-cps@2.27">
     <script>node {
-    stage ('build') {
+    stage ('set version number') {
         git  '{insert-git-url}'
+        def userInput = input (
```

Comment:
I suggest using the build number ${env.BUILD_NUMBER} rather than prompting the user. 

No one wants to deal with versioning if it can be done automatically :)